sc/social/like/plugins/interfaces.py

```diff
@@ -17,6 +17,9 @@ def metadata():
     def plugin():
         ''' Return html render method '''
 
+    def link():
+        ''' Return html render method when high privacy is required'''
```

Comment:
I found triple single quote everywhere so I respected the current code style (commonly I use triple double too)